src/libs/actions/App.js

```diff
@@ -48,10 +48,8 @@ function setLocale(locale) {
  * @param {string} url relative URL starting with `/` to open in expensify.com
  */
 function openSignedInLink(url = '') {
-    API.GetAccountValidateCode().then((response) => {
-        const exitToURL = url ? `?exitTo=${url}` : '';
-        const validateCodeUrl = `v/${currentUserAccountID}/${response.validateCode}${exitToURL}`;
```

Comment:
How come there is no more `exitTo` param?

Reply:
The exitTo was there for the validate page to redirect to the page we wanted after validating. Now, we don't need it because we send users directly to the page we want, without any intermediary pages

Reply:
Got it, thanks!